vasquez_a10.js

```diff
@@ -0,0 +1,21 @@
+/*
+Bryan Vasquez
+vasquez_a10.js
+INFO 2124
+Thoendel
+02/23/2020
+*/
+
+
+document.getElementById('action').addEventListener('click', function() {
+    //WRITE YOUR SOLUTION BETWEEN THIS COMMENT
+    let img = document.getElementById('photo'); //Getting element by ID 
+    img.setAttribute('src', '02.jpg'); //then, changing the src for the new picture.

```

Comment:
setting the img to 2! For the EC - you could use an if to check the value of src and then switch it to 1 or 2 based on the if...